### ISSUE TITLE
Expose MNE preprocessing functions

### ISSUE THREAD

**Issue by robintibor** [braindecode/braindecode#476]
Right now we only have this `Preprocessor` object and users have to look into mne what kind of preprocessing functions exist and what their arguments are.

I propose, that for some selected functions, like resampling pick channels etc, we try to automatically expose them in braindecode by:
1. Create just a list like ['resample', 'pick_channels'] etc into `preprocessing/__init__.py` or somewhere.
2. Automatically read this list on import and create classes like `Resample`  that will work as if `Preprocessor(' resample',...)` was called and that automatically have doc strings copied from mne, also exposed in the documentation

So basically, from the outside it should look like we have a `Resample` class with the same arguments as mne that in effect will just behave as `Preprocessor('resample',...) `. But all that should be automatically created on import, so we as Braindecode developers only have to provide a list of mne functions we would like to expose in braindecode.

I assume this should be possible in python, we also have some of similar doc string copying logic in the moabb datasets already, may serve as an inspiration.

**Comment by bruAristimunha** [comment]
@brunaafl
